WebUI/src/lib/components/modals/WiFiDetails.svelte

```diff
@@ -1,21 +1,23 @@
 <script lang="ts">
-	import { WiFiStateStore } from "$lib/stores";
-	import { getModalStore } from "@skeletonlabs/skeleton";
+  import { WiFiStateStore } from '$lib/stores';
+  import { getModalStore } from '@skeletonlabs/skeleton';
 
   export let bssid: string;
 
   const modalStore = getModalStore();
 
-  $: item = $WiFiStateStore.networks.find(i => i.bssid === bssid);
+  $: item = $WiFiStateStore.networks[bssid];
 
-  $: rows = item ? [
-    { key: 'SSID', value: item.ssid },
-    { key: 'BSSID', value: item.bssid },
-    { key: 'Channel', value: item.channel },
-    { key: 'RSSI', value: item.rssi },
-    { key: 'Secure', value: item.secure },
-    { key: 'Saved', value: item.saved },
-  ] : [];
+  $: rows = item
+    ? [
+        { key: 'SSID', value: item.ssid },
+        { key: 'BSSID', value: item.bssid },
+        { key: 'Channel', value: item.channel },
+        { key: 'RSSI', value: item.rssi },
+        { key: 'Security', value: item.security },
+        { key: 'Saved', value: item.saved },
+      ]
+    : [];
 </script>
```

Comment:
On one hand I thought this was crazy syntax, but on the other it looks like you're defining the variable `$` twice, once with `item` type and once with `rows` type?

I'm very confused. If this is some special syntax I don't know about yet, please refer me to docs that explain them. Thanks.

Reply:
$: is declaring a reactive piece of code, its not a typo, its a feature TM
https://learn.svelte.dev/tutorial/reactive-declarations